pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
+  - "./**/src/main/node"
```

Comment:
Any project containing a `src/main/node` directory is considered as a pnpm package.